### ISSUE TITLE
Web component Canary-Content item Horizontal (with thumbnail) - graphic is not identified

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6595]
## Detailed description

Issue:
1. In JAWS with FF, There is no alt text for the graphic
2. In Vo with iPhone, the graphic is not identified

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/820fdd24-74c2-496f-b474-dadf09c25be7)

**Comment by annawen1** [comment]
@Praveen-sr can you check this issue again? Testing with VO on Iphone on Safari browser, I am able to hear the "Image alt text" assigned to the `alt` of the image

**Comment by Praveen-sr** [comment]
@annawen1 Ram re tested this on Web component Canary with both JAWS n VO iPhone, working fine now. Thanks!
cc @RobertaJHahn

**Comment by RobertaJHahn** [comment]
Closing based on Praveen's comments above.
